ocs_ci/deployment/aws.py

```diff
@@ -762,3 +763,69 @@ def destroy_cluster(self, log_level="DEBUG"):
         suffixes = ['inf', 'vpc']
         stack_names = [f'{cluster_name}-{suffix}' for suffix in suffixes]
         self.aws.delete_cloudformation_stacks(stack_names)
+
+
+class AWSUPIFlexy(AWSBase):
+    """
+    All the functionalities related to AWS UPI Flexy deployment
+    lives here
+    """
+    def __init__(self):
+        self.cluster_name = config.ENV_DATA['cluster_name']
```

Comment:
This should be initiated in https://github.com/red-hat-storage/ocs-ci/blob/e876d7d8c15158ff87d145817350b6cbca63546e/ocs_ci/deployment/cloud.py#L32 , 
so no need to do it here again.